src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="block_stars">
+      <div class="stars--0 stars__flex">
```

Comment:
![Screenshot 2023-02-22 at 18 47 46](https://user-images.githubusercontent.com/6526149/220698102-02fa589b-48e1-4364-a0d0-9c878e289c50.png)
```suggestion
      <div class="stars--0 stars">
```